<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="start_game">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>992</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>791</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Доп. инфа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/btn_bottom.png) stretch;</string>
       </property>
       <property name="text">
        <string>О нас</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Че обозревать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>next day</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>180</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="polosi_gazeti">
     <item>
      <widget class="QLabel" name="polosa1">
       <property name="text">
        <string>Полоса 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="polosa2">
       <property name="text">
        <string>Полоса 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="polosa3">
       <property name="text">
        <string>Полоса 3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>bg</zorder>
   <zorder>picture</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>start_game</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>start_game</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>calendarWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>picture</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>polosa1</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>1130</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>polosa2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>1130</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>polosa3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>1130</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
